tests/api/test_search.py

```diff
@@ -57,17 +56,17 @@ def test_invalid_spatial_search():
 
 def test_temporal_search_single_tailed():
     # Test single tailed
-    utcnow = datetime.utcnow().replace(microsecond=0, tzinfo=timezone.utc)
-    utcnow_str = utcnow.strftime(DATETIME_RFC339)
+    utcnow = datetime.now(timezone.utc).replace(microsecond=0, tzinfo=timezone.utc)
+    utcnow_str = utcnow.isoformat()
     search = Search(collections=["collection1"], datetime=utcnow_str)
```

Comment:
IMO this should fail because `utcnow_str` doesn't follow the RFC3339

datetime should either be a datetime object or a valid `RFC3339` string (from my POV)

Reply:
I am not sure what the issue is. `utcnow_str` is compliant with RFC3339 here. 

Fractional seconds are optional in the spec and either `Z` or `("+" / "-") time-hour ":" time-minute`. 

I did realize the whole `.replace()` part is not really necessary, since the old way was doing `strftime` and didn't include those anyways. And now they do not cause problems.